patches/minecraft/net/minecraft/network/play/server/SJoinGamePacket.java.patch

```diff
@@ -1,10 +0,0 @@
---- a/net/minecraft/network/play/server/SJoinGamePacket.java
-+++ b/net/minecraft/network/play/server/SJoinGamePacket.java
-@@ -21,6 +21,7 @@
-    private boolean field_149204_b;
-    private GameType field_149205_c;
-    private GameType field_241785_e_;
-+   private int dimensionInt;
-    private Set<RegistryKey<World>> field_240811_e_;
```

Comment:
This seems incorrect, to be automatically fixed, but yes this int should also be deleted because we no longer use it. Was this and the others below this done manually?

Reply:
Yes, this was from a manual sweep over all the patches to catch little things like this that slipped in during the update process. I should have mentioned that.